src/types.ts

```diff
@@ -1776,3 +1776,20 @@ export type TranslationLanguages =
   | 'vi'
   | 'zh'
   | 'zh-TW';
+
+export type RateLimitsInfo = {
+  limit: number;
+  remaining: number;
+  reset: number;
+};
+
+export type RateLimitsMap = {
+  [key: string]: RateLimitsInfo;
+};
+
+export type GetRateLimitsResponse = {
+  android?: RateLimitsMap;
+  ios?: RateLimitsMap;
+  server_side?: RateLimitsMap;
+  web?: RateLimitsMap;
```

Comment:
so we have all these different rate limits per each endpoint? are some of them omitted from response or will be just set to default?

Reply:
By default the endpoint returns all limits for all endpoints and all platforms (web, android, ios, server-side); it can be narrowed to just some of the platforms (with the booleans) and/or specific endpoints (with the `string[]` endpoints param)